### ISSUE TITLE
[Discussion] Non-trivial business functionality

### ISSUE THREAD

**Issue by bkompis** [PA165-MushroomHunter/MushroomHunter#70]
From the milestone 2 requirements, we need "at least 2 non-trivial business functions on Service Layer".

What do we want to do as these?

**Comment by bkompis** [comment]
Ideas from discussion on 14.11.:
  - find which mushrooms were found in a forest
  - which mushrooms grow in the date interval
  - for a given mushroom, find in which forest it was found the most times
  - (find a forest with the least amount of recent visits)

**Comment by bohdancvejn** [comment]
dalsi navrh na netrivialni funkci (v pripade, ze nekdo doda popis visitu..)
napr.
We parked at a Citadel parking place. After 5 minutes of walking we found @E Hrib and right after another ten minutes, we saw @P Muchomurka.

@E - eatable NameOfTheMushroom
@U - uneatable NameOfTheMushroom
@P - poisonous NameOfTheMushroom

tak se nam automaticky popridavaji automaticky chybejici houby do naseho katalogu hub
